draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -109,17 +109,13 @@ Host: example.org
 </artwork>
 
     <t>
-      if the query parameters extend to several kilobytes or more of
-      data it may not be, because many implementations place limits on their size. Often these limits are not
-      known or discoverable ahead of time, because a request can pass through many uncoordinated
-      systems. Additionally, expressing some data in the target URI is inefficient, because it
-      needs to be encoded to be a valid URI.
-    </t>
-
-    <t>
-      Encoding query parameters directly into the request URI also effectively
-      casts every possible combination of query inputs as distinct
-      resources. Depending on the application, that may not be desirable.
+      However, for a query with parameters that extend to several kilobytes or more of
+      data, encoding it in the request URI may not be the best option because 
+      <ul>
+        <li>many implementations place limitations on query size (and those limits are not discoverable ahead of time),</li>
```

Comment:
```suggestion
        <li>many implementations place limitations on query size that are not discoverable ahead of time,</li>
```

Reply:
Thank you! I am incorporating this into the next version (about to be posted!).